aiohttp/resolver.py

```diff
@@ -26,6 +27,15 @@ def __init__(self, loop=None):
         self._loop = loop
 
     async def resolve(self, host, port=0, family=socket.AF_INET):
+        if family == socket.AF_UNSPEC:
+            hosts = await self._resolve(host, port, family=socket.AF_INET)
```

Comment:
1. Why not to pass AF_UNSPEC downto `getaddrinfo` ? Your case will do two lookups instead of one.
2. If previous is not possible, why not ti use `asyncio.gather` to resolve in parallel ?